dev-proxy/ProxyEngine.cs

```diff
@@ -578,4 +577,10 @@ Task OnCertificateSelection(object sender, CertificateSelectionEventArgs e)
         // set e.clientCertificate to override
         return Task.CompletedTask;
     }
+    
+    private void LogHotkeysInstructions()
+    {
+        _logger.LogInformation("Hotkeys: (r)ecord, (s)top recording, (c)lear screen");
```

Comment:
If in the future we add the ability to configure different loggers, then we still want these messages to be shown in the terminal. Let's use `Console.WriteLine` rather than the logger.

Reply:
Got it. Updated